open-models/serving/cloud_run_ollama_gemma2_rag_qa.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>Can you please also include a POST request example using Python code? E.g. using <code>requests.post()</code>.</p><p>The reason I ask is b/c you have curl, langchain, but I think Python is just oddly missing, and calling the model is the most exciting part.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Done, thanks!